.github/workflows/test.yml

```diff
@@ -0,0 +1,39 @@
+name: CI
```

Comment:
Long term I don't think we need this; we already have a workflow which does testing and should be doing these tests.